.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
       after_success:
         - travis-cargo doc-upload
     - os: linux
-      rust: 1.13.0
+      rust: 1.15.0
```

Comment:
How come this was required?

Reply:
It's described in the commit. We need `Arc::weak_count`

Reply:
Ah sorry, missed that. Could this use `get_mut` to avoid the version bump here?

Reply:
https://github.com/tailhook/futures-rs/commit/fd741a31710d87483bada9390ca69adb85b1d5db

Code looks more complex (we can't check at the same place because of borrow checker, checking before is racy). 

Also, I'm not sure if [cleaning task reference](https://github.com/tailhook/futures-rs/commit/fd741a31710d87483bada9390ca69adb85b1d5db#diff-5d38d21573ff6c00c115f52e230ab8beR127) in on EOS is a good idea.

So is there any reason to not to bump version?

Reply:
Er sorry thought I replied to this...

In any case, there was some consternation upgrading to 1.13.0 recently so I was just wondering if it was "easy" to stick to 1.13.0, but looks like no! That's fine, looks good to me.